semgrep_output_v1.proto

```diff
@@ -289,6 +289,35 @@ message DependencyParserError {
   string text = 3574323;
 }
 
+message Finding {
```

Comment:
Since this breaks the backwards compatibility of the `Finding` proto, what do you think about bumping this to `semgrep_output_v2`?